subgraph/src/Governance.mapping.ts

```diff
@@ -1,38 +1,24 @@
-import { Address, BigInt, ethereum } from "@graphprotocol/graph-ts";
-import { governanceDeploymentInitiatives } from "../addresses";
+import { BigInt } from "@graphprotocol/graph-ts";
 import {
   AllocateLQTY as AllocateLQTYEvent,
   ClaimForInitiative as ClaimForInitiativeEvent,
   DepositLQTY as DepositLQTYEvent,
-  Governance as GovernanceContract,
   RegisterInitiative as RegisterInitiativeEvent,
   SnapshotVotesForInitiative as SnapshotVotesForInitiativeEvent,
   UnregisterInitiative as UnregisterInitiativeEvent,
   WithdrawLQTY as WithdrawLQTYEvent,
 } from "../generated/Governance/Governance";
 import { GovernanceAllocation, GovernanceInitiative, GovernanceStats, GovernanceUser } from "../generated/schema";
 
-function initialize(block: ethereum.Block): void {
-  // Initial governance initiatives passed to the constructor (no event emitted)
-  for (let i = 0; i < governanceDeploymentInitiatives.length; i++) {
-    let initiative = new GovernanceInitiative(governanceDeploymentInitiatives[i]);
-    initiative.registeredAt = block.timestamp;
-    initiative.registeredAtEpoch = BigInt.fromI32(1);
-    initiative.registrant = Address.zero();
-    initiative.save();
-  }
-
+function initializeStats(): GovernanceStats {
   // Create initial stats
   let stats = new GovernanceStats("stats");
   stats.totalLQTYStaked = BigInt.fromI32(0);
+  stats.totalOffset = BigInt.fromI32(0);
   stats.totalInitiatives = 0;
```

Comment:
Just noticed `totalInitiatives` is never increased. Is the idea to track the number of GovernanceInitiative entities (i.e. only increased, never decreased, not even on unregistration)?

Reply:
No the idea was to track the number of active initiatives, this is something that was missing.